lib/config/error-codes/index.js

```diff
@@ -130,18 +130,28 @@ function checkDomains(errorCodesFiles) {
       `Error configuration file : Field 'code' must be an integer. (domain: '${domainName}').`);
     assert(
       domain.code >= 0x00 && domain.code <= 0xFF,
-      `Invalid error configuration file : Field 'code' must be between 0 and 255. (domain: '${domainName}').`);
+      `Error configuration file : Field 'code' must be between 0 and 255. (domain: '${domainName}').`);
 
     checkSubdomains(domain);
   }
 }
 
+function loadPluginsError (pluginName, pluginErrors) {
+  plugins.subdomains[pluginName] = {};
+  plugins.subdomains[pluginName].code = Object.keys(plugins.subdomains).length;
```

Comment:
We should start plugins custom subdomain code at something like 10 so in the futur we could still add subdomains to plugin without breaking existing plugins